workflows/general/bam-to-fastqs.wdl

```diff
@@ -43,27 +43,27 @@ workflow bam_to_fastqs {
     input {
         File bam
         String pairing = "Paired-end"
-        Int samtools_sort_ncpu = 1
-        Int bam_to_fastq_memory_gb = 40
-        Int max_retries = 1
         Boolean detect_nproc = false
+        Int max_retries = 1
     }
 
     parameter_meta {
         bam: "BAM file to split into fastqs."
-        samtools_sort_ncpu: "Number of CPUs to use while sorting the BAM."
-        bam_to_fastq_memory_gb: "How much memory to provide while converting to fastqs."
         max_retries: "Maximum number of times to retry on a failure."
     }
 
     call samtools.quickcheck { input: bam=bam, max_retries=max_retries }
-    call samtools.split { input: ncpu=samtools_sort_ncpu, bam=bam, max_retries=max_retries, detect_nproc=detect_nproc }
+    call samtools.split { input: bam=bam, detect_nproc=detect_nproc, max_retries=max_retries }
     scatter (split_bam in split.split_bams) {
-        call picard.bam_to_fastq { input: bam=split_bam, memory_gb=bam_to_fastq_memory_gb, max_retries=max_retries }
```

Comment:
Can this be set directly at runtime? I assume so, but I don't recall for subworkflows if that works.

Reply:
Yes it can be set at runtime.